BT-Tracking/Services/Configuration.swift

```diff
@@ -15,37 +15,53 @@ struct Configuration {
 
     let showExposureForDays = 10
 
-    let healthAuthority = "cz.covid19cz.erouska.dev"
-
-    let uploadURL = URL(string: "https://exposure-i5jzq6zlxq-ew.a.run.app/v1/publish")!
+    var healthAuthority: String {
+        #if PROD
+        return "cz.covid19cz.erouska.dev"
+        #else
+        return "cz.covid19cz.erouska.dev"
+        #endif
+    }
 
-    let downloadsURL = URL(string: "https://storage.googleapis.com/exposure-notification-export-ejjud/")!
+    var uploadURL: URL {
+        #if PROD
+        return URL(string: "https://exposure-fghz64a2xa-ew.a.run.app/v1/publish")!
+        #else
+        return URL(string: "https://exposure-i5jzq6zlxq-ew.a.run.app/v1/publish")!
+        #endif
+    }
 
-    let verificationURL = URL(string: "https://apiserver-eyrqoibmxa-ew.a.run.app")!
+    var downloadIndexName: String {
+        #if PROD
+        return "index.txt"
+        #else
+        return "/index.txt"
```

Comment:
Tady to chces aspon stejny format. Tj je to cesta nebo jen nazev souboru?

Reply:
Je to blby, puvodne to tak vubec nebylo, ale ten dev server ma tam jeste jedno lomitko pred a tak to funguje bez zmen v kodu.